lib/travis/build/script/common_lisp.rb

```diff
@@ -0,0 +1,63 @@
+# Maintained by:
+# Thayne McCombs  @tmccombs <astrothayne@gmail.com>
+# Luis Oliveira   @luismbo  <luismbo@gmail.com>
+# SANO Masatoshi  @snmsts   <snmsts@gmail.com>
+# Stelian Ionescu @sionescu <sionescu@cddr.org>
+
+require 'tempfile'
+
+module Travis
+  module Build
+    class Script
+      class CommonLisp < Script
+        DEFAULTS = {
+          lisp: 'sbcl'
+        }
+
+        ROS_URL = "https://raw.githubusercontent.com/roswell/roswell/release/scripts/install-for-ci.sh"
+
+        SYSTEM_MISSING = %Q[\
+No test script provided.
+Please either override the script: key
+]
+
+        def configure
+          super
+          sh.echo "If you encounter any issues with Common Lisp support, please Cc @tmccombs, @luismbo, @snmsts or @sionescu."
```

Comment:
I know the verbiage is not completely consistent, but could you take a look at other community-supported languages such as https://travis-ci.org/Vauxoo/runbot_branch_remote_name_grp_feature2/jobs/110692774#L1399 to mention GitHub Issues link?

Thanks.